server/plugin/virtual_agent.go

```diff
@@ -76,15 +77,46 @@ type TopicPickerControl struct {
 }
 
 type OutputCard struct {
-	UIType string `json:"uiType"`
-	Group  string `json:"group"`
-	Data   string `json:"data"`
+	UIType       string `json:"uiType"`
+	Group        string `json:"group"`
+	Data         string `json:"data"`
+	TemplateName string `json:"templateName"`
+}
+
+type OutputCardRecordData struct {
+	SysID            string         `json:"sys_id"`
+	Subtitle         string         `json:"subtitle"`
+	DataNowSmartLink string         `json:"dataNowSmartLink"`
```

Comment:
just a question the casing for json variables is different there is snake case for dataNowSmartLink and not for other variables.

Reply:
yes, we are getting different casing in the response